package.json

```diff
@@ -11,18 +11,28 @@
     "vscode": "^1.44.0"
   },
   "activationEvents": [
-    "onLanguage:mint"
+    "onLanguage:mint",
+    "onCommand:mint.build",
+    "onCommand:mint.compile",
+    "onCommand:mint.docs",
+    "onCommand:mint.formatAll",
+    "onCommand:mint.init",
+    "onCommand:mint.install",
+    "onCommand:mint.loc",
+    "onCommand:mint.start",
+    "onCommand:mint.test",
+    "onCommand:mint.version"
   ],
   "main": "./out/src/extension",
   "devDependencies": {
     "@types/node": "^13.13.4",
-    "@types/vscode": "^1.44.0",
     "js-yaml": "^3.13.1",
-    "typescript": "^3.8.3"
+    "typescript": "^3.8.3",
+    "vscode": "^1.1.37"
```

Comment:
Let's remove the `vscode` dependency, it brings in security vulnerabilities.

Reference: https://code.visualstudio.com/updates/v1_36#_splitting-vscode-package-into-typesvscode-and-vscodetest

Reply:
Here is how it worked earlier:
https://github.com/s0kil/mint-vscode/commit/7089de5ffd6936ad7b421bfeae5c4df08f9f78b4

Reply:
Thanks for the explanation, I have resolved it